### PR TITLE
Update variable names

### PR DESCRIPTION
**Pull request by wholmgren**
Lots of API changes going into this PR. Not ready yet, just getting it going early since it relates to #69.

to do:
- [x] snlinverter
- [x] sapm outputs
- [x] calcparams_desoto kwargs and doc string
- [x] absoluteairmass arg
- [x] relativeairmass arg
- [x] ineichen output
- [x] haurwitz arg and output
- [x] lots in irradiance.py

### CODE REVIEW

**Comment by wholmgren** [comment]
I changed some variable names that we hadn't yet discussed, so commenters in #37 may want to look at the changes to ``singlediode``.

**Comment by wholmgren** [comment]
I'm sure I missed some things, but this now goes most of the way to closing #37. This is a good time for comments/requests.

I'll see about updating the tutorials tomorrow. @NelisW updated the tutorials in his fork, and I want to look at that too. Sure would be nice if GitHub could show better notebook diffs. 

The TravisCI fails are all due to #72. Everything works on 2.7.

File: pvlib/clearsky.py [hunk 1]
```diff
@@ -195,7 +195,9 @@ def ineichen(time, location, linke_turbidity=None,
     
     cos_zenith = tools.cosd(ApparentZenith)
     
-    clearsky_GHI = cg1 * I0 * cos_zenith * np.exp(-cg2*AMabsolute*(fh1 + fh2*(TL - 1))) * np.exp(0.01*AMabsolute**1.8)
+    clearsky_GHI = ( cg1 * I0 * cos_zenith *
```
Comment (bmu) [inline pvlib/clearsky.py]: would also change `clearsky_GHI` to all lower case (`clearsky_ghi`) for consistency.
Not sure, did you intended to change only names for the kw arguments (all api related) or all internal variable names too?

**Comment by NelisW** [comment]
Trying my best to get the notebooks working.  Two remaining (pvsystem.ipynb, Test_Scripts_1.ipynb) that need some attention.  The refactoring broke some of the notebooks at quite a deep level.  Hope to spend some time again tonight.  In this part of the world we have semi-regular load shedding - was down again last night for 2 hours.  You can work on a laptop, but have no internet connection - makes you realise how dependent we have become on the internet.

**Comment by wholmgren** [comment]
@NelisW I made some updates to the pvsystem notebook in this PR since I wanted to plot some things to make sure that it was all ok. Test_Scripts_1 and TS_to_irradiance haven't been touched in a long, long time.

File: pvlib/clearsky.py (hunk 1 above)
Comment (wholmgren) [inline pvlib/clearsky.py]: I only intended to change the API related names for now. I would like to see the internal variables get updated, but I think we can do this in smaller chunks.

**Comment by wholmgren** [comment]
The tests now pass by forcing the use of numpy 1.8 and scipy 0.14 on Python 3. You can install any of the pandas versions on top if this if you use ``--no-deps``. I will update the readme separately.

**Comment by wholmgren** [comment]
@bmu do you think we need to update all of the internal variables in this PR or can we deal with that later?

**Comment by bmu** [comment]
No Problem to deal with it later. 

Am 29. Juni 2015 17:50:54 MESZ, schrieb Will Holmgren <notifications@github.com>:

**Comment by wholmgren** [comment]
@bmu @Calama-Consulting I reviewed this again and I think it's ready to merge if you're happy with the changes.

**Pull request merged**